package.json

```diff
@@ -40,7 +40,8 @@
   "dependencies": {
     "@ant-design/icons": "^4.5.0",
     "antd": "^4.13.0",
-    "lodash": "^4.17.21"
+    "lodash": "^4.17.21",
+    "moment": "^2.0.0"
```

Comment:
Big nope, see https://momentjs.com/docs/#/-project-status/. Use native date or https://date-fns.org/.

Reply:
Thanks for the recommendation about date-fns! I've updated the PR with this lib...
Here's the visual result onHover:
![image](https://user-images.githubusercontent.com/26680884/115671713-17acb000-a34b-11eb-9935-968b00716dde.png)